src/components/form-elements/_form-elements.scss

```diff
@@ -144,13 +150,13 @@ $include-html: false !default;
     &--valid {
       border-color: $inputValidBorderColor;
       &:focus {
-        border-color: $inputValidBorderColor;
+        border-color: $inputFocusValidBorderColor;
```

Comment:
Unrelated change:  color of the border for 'valid' and 'invalid' fields changes on focus (it wasn't changing before)